lua/lazyvim/plugins/editor.lua

```diff
@@ -158,7 +158,6 @@ return {
       { "<leader>fr", "<cmd>Telescope oldfiles<cr>", desc = "Recent" },
       { "<leader>fR", LazyVim.telescope("oldfiles", { cwd = vim.uv.cwd() }), desc = "Recent (cwd)" },
       -- git
-      { "<leader>gc", "<cmd>Telescope git_commits<CR>", desc = "Commits" },
```

Comment:
If you'd like to not remove a mapping, `<leader>gl` is free and makes sense to me.

Reply:
But they do the same thing. They show the commits. It doesn't make sense to have 2 different mappings for 2 plugins that do the same thing essentially (to me personally at least).

Reply:
> But they do the same thing. They show the commits. It doesn't make sense to have 2 different mappings for 2 plugins that do the same thing essentially (to me personally at least).

I agree.